driver.js

```diff
@@ -447,7 +447,7 @@ class Driver {
   async open(url, headers = {}, storage = {}) {
     const site = new Site(url.split('#')[0], headers, this)
 
-    if (storage.local || storage.session) {
+    if (true) {
```

Comment:
Waarom blijft het 'if'-statement hier staan, is het alleen om tijdelijk `storage.local / storage.session` te skippen?

Reply:
Om de diff kleiner te houden. storage.local en session zijn allebei objecten, dus altijd truthy, dus nutteloze if. Had helemaal weggekund, maar dan was de diff onleesbaar en dat is irritant tijdens dev, maar kan nu wel.